build_deploy.sh

```diff
@@ -6,5 +6,7 @@ source 'scripts/deploy/build-deploy-common.sh'
 
 IMAGE_NAME="${IMAGE_NAME:-quay.io/cloudservices/compliance-backend}"
 ADDITIONAL_TAGS="latest"
+REQUIRED_REGISTRIES_LOCAL=''
+DISABLE_BUILD_CACHE='true'
```

Comment:
A small suggestion to be consistent with other variables/options.

```suggestion
DISABLE_BUILD_CACHE="true"
```

Reply:
@Victoremepunto, do we want to add a small comment over this variable that explains why we are disabling the build cache? For some people, it might be a surprise as build-cache is often cited as a good practice.

I also wonder if having a small helper called `runtime_is_docker()` (or something along these lines) to detect Docker vs Podman and re-enable support for build-cache should be the former.

Reply:
@kwilczynski  I pushed a draft version based on your suggestion - a detection mechanism that prevents explicit cache disabling configuration, but rather automatic detection if the engine is `podman` and the version is under `4.5.0` - please , take a look and let me know to move forward (I can undraft if it seems okay)